src/app/ipv/middleware.js

```diff
@@ -371,63 +326,16 @@ module.exports = {
     try {
       checkForIpvAndOauthSessionId(req, res);
 
-      if (req.body?.journey === "end") {
-        await handleJourneyResponse(req, res, "end", currentPageId);
-      } else if (req.body?.journey === "addressCurrent") {
-        await handleJourneyResponse(req, res, "address-current", currentPageId);
-      } else if (req.body?.journey === "attempt-recovery") {
-        await handleJourneyResponse(
-          req,
-          res,
-          "attempt-recovery",
-          currentPageId,
-        );
-      } else if (req.body?.journey === "build-client-oauth-response") {
-        req.session.ipAddress = req?.session?.ipAddress
-          ? req.session.ipAddress
-          : getIpAddress(req);
-        await handleJourneyResponse(
-          req,
-          res,
-          "build-client-oauth-response",
-          currentPageId,
-        );
-      } else {
-        await handleJourneyResponse(req, res, "next", currentPageId);
-      }
-    } catch (error) {
-      transformError(error, `error handling POST request on ${currentPageId}`);
-      next(error);
-    }
-  },
+      req.session.ipAddress = req?.session?.ipAddress
+        ? req.session.ipAddress
+        : getIpAddress(req);
 
-  handleMultipleDocCheck: async (req, res, next, currentPageId) => {
-    try {
-      checkForSessionId(req, res);
-
-      if (req.body?.journey === "next/passport") {
-        await handleJourneyResponse(req, res, "ukPassport", currentPageId);
-      } else if (req.body?.journey === "next/driving-licence") {
-        await handleJourneyResponse(req, res, "drivingLicence", currentPageId);
-      } else {
-        await handleJourneyResponse(req, res, "end", currentPageId);
```

Comment:
The default case here is `end` but as far as I can see with the new code the default will be `next` is that deliberate?
Same in `handleEscapeM2b` and `handleEscapeAction`

Reply:
Ah, I think the case should probably be that there isn't a default then. I think in the use cases I've seen, the "next" or "end" event is actually provided in the `req.body?.journey`